### PR TITLE
Card Groupings Subsystem

### PR DESCRIPTION
**Pull request by jliburd2167**
Added Card class for managing the cards in the game. Currently not included in the database but can update to be included in the database in required for persistent data storage during sessions.

Added Deck class to manage player decks
Created Hand class to manage a stack of cards in the a players hand. Added pytest to conda environment
Updated Read me with instructions on running pytest

### CODE REVIEW

**Review by jchien1352 — APPROVED** [review]
Will there be a mechanism for checking if a particular String is in the Suspects or Rooms or Weapons sets? Like if "Colonel Orchid" is a valid suspect or not.

File: Backend/cardGroupings/tests/test_Card.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# tests/test_card.py
+import pytest
+from Card import CardType, Card
+
```
Comment (abisho21) [inline Backend/cardGroupings/tests/test_Card.py:4]: Django has testing library that is built upon pytest that allows one to test the entire Django suite e.g. (Models, Views, Authentication, Channels). Furthermore, you can create a base class that inherits the unit.TestCase class that allows one to have a setup function and tear down function for setting up the test cases.

**Review by abisho21 — APPROVED** [review]
Django has testing library that is built upon pytest that allows one to test the entire Django suite e.g. (Models, Views, Authentication, Channels). Furthermore, you can create a base class that inherits the unit.TestCase class that allows one to have a setup function and tear down function for setting up the test cases.

**Comment by jliburd2167** [comment]
> Will there be a mechanism for checking if a particular String is in the Suspects or Rooms or Weapons sets? Like if "Colonel Orchid" is a valid suspect or not.

Good catch. Added Validity checks and test cases for valid names and card type combo.

**Pull request merged**